images/bay-php/entrypoints/bay-shared-temp-files-dir-check.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/sh
-set -euo pipefail
+# set -euo pipefail
```

Comment:
What is this commented out for @steveworley ?

Reply:
Yeah, that's why — I thought I did put a comment to that effect somewhere. We can probably toggle these options like the kms decrypt script is doing, but I wasn't sure exactly for this one.